### PR TITLE
Cleanup kotlinx.html publishing configuration

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: .teamcity/src/subprojects/VcsRoots.kt [hunk 1]
```diff
@@ -45,12 +45,16 @@ object VCSAPIDocs : PasswordVcsRoot({
 object VCSKotlinxHtml : PasswordVcsRoot({
     name = "Kotlinx.html Library"
     url = "https://github.com/Kotlin/kotlinx.html.git"
+    branchSpec = """
```
Comment (cy6erGn0m) [inline .teamcity/src/subprojects/VcsRoots.kt:48]: Don't we already have it somewhere? Or we simply copy-paste it over roots?

File: .teamcity/src/subprojects/kotlinx/html/PublishKotlinxHtmlToCentral.kt [hunk 2]
```diff
@@ -10,36 +10,27 @@ import subprojects.release.*
...
-        features {
-            perfmon { }
-        }
-
         params {
             configureReleaseVersion()
+            defaultTimeouts()
+            password("env.SIGN_KEY_PASSPHRASE", value = "%sign.key.passphrase%")
+            password("env.SIGN_KEY_PRIVATE", value = "%sign.key.private%")
+            password("env.PUBLISHING_USER", value = "%sonatype.username%")
```
Comment (cy6erGn0m) [inline .teamcity/src/subprojects/kotlinx/html/PublishKotlinxHtmlToCentral.kt:26]: I wonder if we have rights for publishing to the  Kotlinx group from this account?

**Review by cy6erGn0m — APPROVED** [review]

File: .teamcity/src/subprojects/VcsRoots.kt (hunk 1 above)
Comment (e5l) [reply .teamcity/src/subprojects/VcsRoots.kt:48]: copypaste

File: .teamcity/src/subprojects/kotlinx/html/PublishKotlinxHtmlToCentral.kt (hunk 2 above)
Comment (e5l) [reply .teamcity/src/subprojects/kotlinx/html/PublishKotlinxHtmlToCentral.kt:26]: we will check :)

**Pull request merged**